CONTRIBUTING.md

```diff
@@ -11,28 +11,28 @@ Please note that if you open a bug report and your issue does not follow our tem
 
 ## Contributing to the source code
 
-0. Fork this repo, and then clone it:
```

Comment:
We actually do want this to point to the users username, not `chaoss/augur`, as its supposed to point to a fork.

Reply:
Thanks for the clarification @sgoggins! I'll also consider this for new PR's.

Reply:
I updated the pr with your correction @sgoggins :)